configuration/globalproperties/additional_clinical_gp.xml

```diff
@@ -0,0 +1,29 @@
+<config>
+  <globalProperties>
+    <globalProperty>
+      <property>emrapi.sqlSearch.noVitalsPatients</property>
+      <value>select distinct concat( ifnull(pn.family_name,'') ,' ', pn.given_name ) as name, 
+          pi.identifier as identifier, 
+          concat("",p.uuid) as uuid, 
+          concat("",v.uuid) as activeVisitUuid, 
+          IF(va.value_reference = "Admitted", "true", "false") as hasBeenAdmitted 
+        from visit v 
+        join person_name pn on v.patient_id = pn.person_id and pn.voided = 0 
+        join patient_identifier pi on v.patient_id = pi.patient_id 
+        join patient_identifier_type pit on pi.identifier_type = pit.patient_identifier_type_id 
+        join global_property gp on gp.property="bahmni.primaryIdentifierType" and gp.property_value=pit.uuid 
+        join person p on p.person_id = v.patient_id
+        join location l on l.uuid = ${visit_location_uuid} and v.location_id = l.location_id 
+        left outer join visit_attribute va on va.visit_id = v.visit_id and va.attribute_type_id = ( 
+          select visit_attribute_type_id from visit_attribute_type where name="Admission Status" 
+        ) and va.voided = 0 
+        where v.date_stopped is null AND v.voided = 0 and v.patient_id not in (
+          select distinct person_id from obs
+          join encounter on encounter.encounter_id = obs.encounter_id
+          join visit on encounter.visit_id = visit.visit_id
+          where visit.date_stopped is null and obs.concept_id = (select concept_id from concept where uuid = "415cc6cb-5666-492d-b775-bc668a8be3b9") 
```

Comment:
Where does this UUID come from?
I think the Haiti Vitals Signs UUID is not hardcoded in the CSV. https://github.com/mekomsolutions/openmrs-config-haiti/blob/009d8c312aca01bf764b3ff615890d22a48720a9/configuration/concepts/vitals_form.csv#L35
If you want to refer to this concept via UUID (not FSN) then a UUID should must be hardcoded in Haiti Config.

Reply:
That's true, the UUID is not hardcoded in the [Haiti repo](https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/vitals_form.csv). I'll change to using the FSN.

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/6/commits/87ec2f0ccc4b2289023e264352aaec7879b60a36)

Reply:
@rubailly , best practice is to avoid using FSNs as a reference to a concept. I have updated the OpenMRS Config Haiti to always specify UUIDs.
You can use `773d36e9-119e-452c-91ee-4ac90245e504` to refer to `Haiti_Vital Signs Form` .

Reply:
[Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/6/commits/4e9e2851700b734182e366a1aa76564b71db76e2)